src/App.js

```diff
@@ -8,9 +8,11 @@ import React from 'react';
 import logo from './logo.png';
 import './App.css';
 import Dashboard from './components/Dashboard';
+import TopBar from './components/TopBar';
 
 const App = () => (
   <div className="App">
+    <TopBar />
     <header className="App-header">
       <img src={logo} className="App-logo" alt="logo" />
```

Comment:
Could you please remove the huskyCI dashboard logo from our dashboard? With the top bar, we could keep it simple and just show the data from the dashboard themselves.

Reply:
Done.